repl/tests/tests.rs

```diff
@@ -60,3 +64,112 @@ fn issue_365_run_io_from_command_line() {
     }
     assert_eq!(String::from_utf8_lossy(&output.stdout), "123\n");
 }
+
+
+static TIMEOUT: u64 = 10_000;
+static PROMPT: &str = ">";
+
+#[test]
+fn prompt() {
+    let mut repl = spawn("../target/debug/gluon -i", Some(TIMEOUT)).unwrap();
+    repl.exp_string(PROMPT).unwrap();
```

Comment:
The error messages seems rather bad when using unwrap. Perhaps wrap the tests in a result returning function and do `unwrap_or_else(|err| panic!("{}", err))` on that function. Or add a macro like

```rust
macro_rules! t {
    ($e: expr) => { $e.unwrap_or_else(|err| panic!("{}", err)) }
}
t!(repl.exp_string(PROMPT))
```

Reply:
Agreed about unwrap! That macro will be very useful.